syntax = "proto3";

package ArmoniK.api.grpc.v1.ResultStatus;

option csharp_namespace = "ArmoniK.Api.gRPC.V1";


enum ResultStatus
{
  RESULT_STATUS_UNSPECIFIED = 0;
  RESULT_STATUS_CREATED = 1;
  RESULT_STATUS_COMPLETED = 2;
  RESULT_STATUS_ABORTED = 3;
  RESULT_STATUS_NOTFOUND = 9;
}
